README.md

```diff
@@ -1,2 +1,73 @@
 # tekton-example
-Example implementation of a tekton pipeline that deploys an Appsody project.
+Example implementation of a Tekton pipeline that deploys an Appsody project.
+
+## Introduction
+This repo contains an example of a Tekton pipeline that builds and deploys an application created with [Appsody](https://github.com/appsody/appsody) to a Kubernetes cluster. The application is deployed as a Knative Serving service. 
+
+## Prerequisites and assumptions
+This example can be put to fruition once you have fulfilled the following prerequisites:
```

Comment:
I'd like to lose the wording "This example can be put to fruition" perhaps... "In order to run this example, the following prerequisites are required:" or similar?

Reply:
Done.